frontend/src/components/big popup/index.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Drawer } from '@mui/material';
+import Box from '@mui/material/Box';
+
+export default function BigPopup(props: any) {
+  const { isOpen, togleClose } = props;
+
+  const fon: any = true;
+
+  return (
+    <div>
+      <Drawer
+        sx={{ height: 100 }}
+        anchor="left"
+        hideBackdrop={fon}
+        open={isOpen}
+        onClose={togleClose}
+      >
+        <Box sx={{ width: 400, backgroundColor: '#757ce8', mt: 50 }}>
```

Comment:
all colors we should get from theme
For example:  _theme.palette.primary.main_
Use seperate style for it or just import theme using useTheme hook from mui